Sources/SchemaStructure.swift

```diff
@@ -0,0 +1,5 @@
+public struct SchemaStructure {
```

Comment:
As mentioned above, we don't really need another type for this, we can just tuple the metadata with the structure.